src/lib/serviceRoutines/postUpdateContext.cpp

```diff
@@ -312,11 +312,20 @@ static bool updateForward(ConnectionInfo* ciP, UpdateContextRequest* upcrP, Upda
   else  // NGSIv2
   {
     LM_T(LmtForward, ("upcrP->providerFormat == V2. out: '%s'", out.c_str()));
-    // NGSIv2 forward - no payload to be received, just a 204 No Content HTTP Header
-    if (strstr(out.c_str(), "204 No Content") != NULL)
+    // NGSIv2 forward - no payload to be received
+
+    //if (strstr(out.c_str(), "204 No Content") != NULL)
```

Comment:
We cannot use string matching for this. Note that reason phrases such as "No Content" or "Not Found", although used 
by most implementations, are not mandatory from a formal point of view. From the HTTP spec at https://www.w3.org/Protocols/rfc2616/rfc2616-sec6.html: _"The reason phrases listed here are only recommendations -- they MAY be replaced by local equivalents without affecting the protocol"_.

It seems I left a leftover comment. Pending to be removed.

Reply:
Fixed in 2164edf